C_functions/test_C.c

```diff
@@ -0,0 +1,25 @@
+#include <Python.h>
+#include <string.h>
+static PyObject *
+test_C(PyObject *self, PyObject *args)
+{
+    const char *lista;
+    int sts;
+
+    if (!PyArg_ParseTuple(args, "s", &lista))
+        return NULL;
+    sts=system(lista);
```

Comment:
calling system() is dangerous, lets just print this instead of running it (even though it's just an example)